templates/taxi/manufacturer_list.html

```diff
@@ -1,6 +1,11 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" class="btn-primary" value="🔍">
```

Comment:
Consider adding the `btn` class to the submit button for consistent styling with other buttons, i.e., `class="btn btn-primary"`.